02-peer-protocol.md

```diff
@@ -94,6 +94,7 @@ desire to set up a new channel.
    * [`4`:`feerate_per_kw`]
    * [`2`:`to_self_delay`]
    * [`2`:`max_accepted_htlcs`]
+   * [`1`:`announce_channel`]
```

Comment:
Could yet another features bitfield be more future-proof? I am fine with a single byte.